dask/dask.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This initialization action script will install Dask, Dask-Yarn and other
+# relevant libraries on a Dataproc cluster. Please see dask.org and 
+# yarn.dask.org for more information.
+
+set -euxo pipefail
+
+readonly DASK_CONFIG_DIR=/etc/dask/
+readonly DASK_CONFIG_FILE=${DASK_CONFIG_DIR}/config.yaml
+
+readonly CONDA_ENV_DIR=/opt/envs
+readonly CONDA_ENV_NAME=dask
+readonly CONDA_ENV_PACK=${CONDA_ENV_DIR}/${CONDA_ENV_NAME}.tar.gz
+
+readonly CONDA_EXTRA_PACKAGES="$(/usr/share/google/get_metadata_value attributes/CONDA_PACKAGES || echo "")"
+readonly CONDA_EXTRA_CHANNELS="$(/usr/share/google/get_metadata_value attributes/CONDA_CHANNELS || echo "")"
+
+readonly ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
+
+mkdir -p ${CONDA_ENV_DIR} ${DASK_CONFIG_DIR}
+
+readonly CONDA_BASE_CHANNELS=(
+    "conda-forge"
+)
+
+readonly CONDA_BASE_PACKAGES=(
+    "dask-yarn=0.8.1"
+    "dask>=2.6.0"
+    "dill"
+    "ipykernel"
+    "fastparquet"
+    "gcsfs"
+    "pyarrow"
+)
+
+function install_conda_kernel() {
+  conda install -y nb_conda_kernels
```

Comment:
Why do we need this?

Reply:
This allows the created conda environment to show up as a kernel option in Jupyter.